chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+output=output
+mkdir -p ${output}
+
+# Create file with base64 string with 700 permission
+# arg1 path
+function createFile {
+  local out=./${output}/$1
+  head -n 10 /dev/urandom | base64 > ${out}
+  chmod 700 ${out}
+}
+
+# Get file name from path
+# arg1 path
+function getFileName {
+  echo "$1"| sed "s|.*\/||"
+}
+
+# Show file content
+# arg1 path to file
+function showFile {
+  echo $1
+  echo ===
```

Comment:
It won't work because `=` is a meta-character. If you want string as it is, quote it with single quotes.
```suggestion
  echo '==='
```

Reply:
Also please use similar quotes within the file `"` or `'`